core/deployment/src/main/java/io/quarkiverse/operatorsdk/deployment/OperatorSDKProcessor.java

```diff
@@ -283,6 +283,21 @@ void ignoreNonIndexedClassesWarningsInReflectiveHierarchy(
         reflectiveHierarchyIgnoreWarningBuildItemBuildProducer.produce(
                 new ReflectiveHierarchyIgnoreWarningBuildItem(
                         DotName.createSimple(io.vertx.core.net.ProxyType.class.getName())));
+        reflectiveHierarchyIgnoreWarningBuildItemBuildProducer.produce(
+                new ReflectiveHierarchyIgnoreWarningBuildItem(
+                        DotName.createSimple("okhttp3.OkHttpClient")));
```

Comment:
@metacosm I don't think this would be worth adding okhttp3 dependency, so I opted to list them as strings.